alt/asyncapi/README.md

```diff
@@ -0,0 +1,81 @@
+asyncapi
+========
+
+The AsyncAPI CLI
+
+[![oclif](https://img.shields.io/badge/cli-oclif-brightgreen.svg)](https://oclif.io)
+[![Version](https://img.shields.io/npm/v/asyncapi.svg)](https://npmjs.org/package/asyncapi)
+[![Downloads/week](https://img.shields.io/npm/dw/asyncapi.svg)](https://npmjs.org/package/asyncapi)
+[![License](https://img.shields.io/npm/l/asyncapi.svg)](https://github.com/asyncapi/cli/blob/master/package.json)
+
+<!-- toc -->
+* [Usage](#usage)
+* [Commands](#commands)
+<!-- tocstop -->
+# Usage
+<!-- usage -->
+```sh-session
+$ npm install -g @asyncapi/cli
+$ asyncapi COMMAND
+running command...
+$ asyncapi (-v|--version|version)
+@asyncapi/cli/0.1.0 darwin-x64 node-v14.15.1
+$ asyncapi --help [COMMAND]
+USAGE
+  $ asyncapi COMMAND
+...
+```
+<!-- usagestop -->
+# Commands
+<!-- commands -->
```

Comment:
List of commands is automatically updated by Oclif when you use their CLI tool.

Reply:
why dont they ever work? one thing i need to know is what is the prompts to check into my home network to see who is in it